static/dbstructure.config.php

```diff
@@ -932,6 +933,7 @@
 			"uid_type_link" => ["uid", "type", "link(190)"],
 			"uri-id" => ["uri-id"],
 			"parent-uri-id" => ["parent-uri-id"],
+			"uid_cid" => ["uid", "cid", "type"],
```

Comment:
Is `type` really needed here? if yes, then please add it to the name of the index as well.

Reply:
Yeah, to filter by type in `Notify->deleteForUserByContactId`